addons/test_mail/tests/test_mail_template.py

```diff
@@ -104,6 +104,14 @@ def test_template_schedule_email(self):
         self.assertFalse(mail.scheduled_date)
         self.assertEqual(mail.state, 'outgoing')
 
+    @mute_logger('odoo.addons.mail.models.mail_mail')
+    def test_template_send_mail_body(self):
+        """ Test that the body is set correctly in 'mail.message' when sending an email from mail.template """
+        self.test_template.send_mail(self.test_record.id)
```

Comment:
It returns the mail.mail ID, so you could browse the mail and check body and body_html content, instead of searching the message